README.md

```diff
@@ -126,7 +346,11 @@ gcloud beta dataflow flex-template run "dlp-s3-scanner-deid-demo" --project=<pro
 --parameters=^~^streaming=true~enableStreamingEngine=true~tempLocation=gs://<path>/temp~numWorkers=5~maxNumWorkers=5~runner=DataflowRunner~filePattern=<s3orgcspath>/filename.csv~dataset=<bq_dataset>~autoscalingAlgorithm=THROUGHPUT_BASED~workerMachineType=n1-highmem-8~inspectTemplateName=<inspect_template>~deidentifyTemplateName=<deid_template>~awsRegion=ca-central-1~awsCredentialsProvider=$AWS_CRED~batchSize=100000~DLPMethod=DEID
 
 ```
-## To Do
-- take out first row as header before processing 
 
+Follow the instructions to [create a flex template](https://cloud.google.com/dataflow/docs/guides/templates/using-flex-templates) and image to avoid running gradle each time.
```

Comment:
Re: Follow the instructions to [create a flex template](https://cloud.google.com/dataflow/docs/guides/templates/using-flex-templates) and image to avoid running gradle each time.

How about: To avoid repeatedly running Gradle, you can [create a flex template](https://cloud.google.com/dataflow/docs/guides/templates/using-flex-templates).